org.eclipse.bluesky/plugin.xml

```diff
@@ -1,25 +1,88 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+
+   <!-- BlueSky/Core -->
    <extension
          point="org.eclipse.core.contenttype.contentTypes">
       <content-type
             base-type="org.eclipse.core.runtime.text"
             id="org.eclipse.bluesky.parent"
             name="BlueSky supported web dev file"
             priority="normal"/>
+   </extension>
+
+   <extension
+         point="org.eclipse.ui.editors">
+      <editorContentTypeBinding
+            contentTypeId="org.eclipse.bluesky.parent"
+            editorId="org.eclipse.ui.genericeditor.GenericEditor">
+      </editorContentTypeBinding>
+   </extension>
+
+   <extension
+         point="org.eclipse.ui.genericeditor.presentationReconcilers">
+      <presentationReconciler
+            class="org.eclipse.tm4e.ui.text.TMPresentationReconciler"
+            contentType="org.eclipse.bluesky.parent">
+      </presentationReconciler>
+   </extension>
+   
+   <extension
+         point="org.eclipse.ui.handlers">
+      <handler
+            class="org.eclipse.bluesky.debug.ToogleLineBreakpointHandler"
+            commandId="org.eclipse.debug.ui.commands.ToggleLineBreakpoint">
```

Comment:
This one is only for JS files so far (although the `enableWhen` is missing), so you should consider moving this to the JS block.